### PR TITLE
Feat/next

### PR DESCRIPTION
**Pull request by hirsch88**
Closes #

https://github.com/baloise/design-system/issues/329
https://github.com/baloise/design-system/issues/323

#### Changelog

**New**

- Select
  - can handle single values instead of an array
  - listen to attribute and content changes 
- Table has his own package now 
- Update dependencies and fixed vulnerabilities
- Updated the Angular and Vue example app
- Improved the documentation

**Changed**

- Checkbox replaced rem with pixel
- Vue renamed `validators` function to `rules`

**Removed**

- Old client libs left overs

### CODE REVIEW

File: examples/vue/src/app/pages/Home.vue [hunk 1]
```diff
@@ -30,7 +20,8 @@ export default defineComponent({
     const count = ref(0)
 
     const onConfirm: onConfirm = (value: string) => {
-      alert(value)
+      console.log('onConfirm', value)
```
Comment (mladenplaninicic) [inline examples/vue/src/app/pages/Home.vue:23]: can you please remove this console log?

File: packages/components/src/components/bal-radio/bal-radio.scss [hunk 2]
```diff
@@ -50,13 +56,9 @@
...
       font-family: $body-family;
 
-      bal-text {
-        padding-left: 32px;
-      }
-
-      bal-text:empty {
-        padding-left: 0;
-      }
+      // bal-text {
```
Comment (mladenplaninicic) [inline packages/components/src/components/bal-radio/bal-radio.scss]: should this be here?

File: packages/components/src/components/bal-select/bal-select.tsx [hunk 3]
```diff
@@ -102,20 +102,51 @@ export class Select {
...
+      if (isString(this.value)) {
+        this.rawValue = [this.value]
+      } else {
+        this.rawValue = [...this.value]
+      }
+    }
+  }
+
+  @Watch('rawValue')
+  rawValueWatcher(newValue: string[], oldValue: string[]) {
```
Comment (mladenplaninicic) [inline packages/components/src/components/bal-select/bal-select.tsx:122]: this too?

File: packages/components/src/components/bal-select/index.html [hunk 4]
```diff
@@ -7,7 +7,7 @@
     <link rel="stylesheet" href="/build/design-system-components.css" />
     <script type="module" src="/build/design-system-components.esm.js"></script>
     <script nomodule src="/build/design-system-components.js"></script>
-    <!-- <script src="/assets/js/events.js"></script> -->
```
Comment (mladenplaninicic) [inline packages/components/src/components/bal-select/index.html]: this too?

File: packages/components/src/components/bal-radio/bal-radio.scss (hunk 2 above)
Comment (hirsch88) [reply packages/components/src/components/bal-radio/bal-radio.scss]: done

File: examples/vue/src/app/pages/Home.vue (hunk 1 above)
Comment (hirsch88) [reply examples/vue/src/app/pages/Home.vue:23]: this is an example app, so it is okay :-)

**Review by mladenplaninicic — APPROVED** [review]

**Pull request merged**